src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -88,6 +88,7 @@ def start_workunit(self, workunit):
         )
       else:
         zipkin_attrs =  create_attrs_for_span(
+          trace_id=self.trace_id, # use UUID from run_id as trace id
```

Comment:
I would probably drop this comment; this code doesn't actually ensure that (code in another file does), and it's not ensured 100% of the time (people can also specify their own trace IDs with a flag)

Reply:
there will be an error if the user will specify only trace_id flag, both trace_id and parent_id should be specified.